python/paddle/fluid/tests/unittests/test_collective_api_base.py

```diff
@@ -203,18 +273,22 @@ def check_with_place(self,
                          static_mode="1",
                          check_error_log=False,
                          need_envs={},
-                         eager_mode=True):
+                         eager_mode=True,
+                         dtype=None):
         if backend == "nccl" or backend == "bkcl":
             with_gloo = '0'
         else:
             with_gloo = '1'
         required_envs = os.environ.copy()
+        dtype = "float32" if dtype is None else dtype
         additional_envs = {
             "NCCL_P2P_DISABLE": "1",
             "STATIC_MODE": static_mode,
             "PADDLE_WITH_GLOO": with_gloo,
+            "PADDLE_DISTRI_BACKEND": backend,
```

Comment:
What's the difference of `PADDLE_DISTRI_BACKEND` and `BACKEND`?

Reply:
It seems like BACKEND is useless, PADDLE_DISTRI_BACKEND is a new environment variable we add. BACKEND is not removed in case, the base test case may be rewrite for it somehow unclear.